data-serving/scripts/converters.py

```diff
@@ -250,7 +250,30 @@ def convert_dictionary_field(
     return {'provided': [value]}
 
 
-def convert_imported_case(id: str, values_to_archive: Series) -> Dict[str, Any]:
+def convert_revision_metadata_field(data_moderator_initials: str) -> Dict[
```

Comment:
Adding date/notes later?

Reply:
Unfortunately, I don't think we can get these fields from the current data :\ It doesn't seem like the date that a row was added to the dataset was recorded. Tomorrow I'm going to see if we can get it added to the current tool for data going forward -- otherwise it will be hard to get many of Steven's metrics (e.x. last day data was added for a given country).

Notes is mostly to support revisions (per the PRD).